fs/src/lib.rs

```diff
@@ -20,23 +18,23 @@ use std::{
     path::{Path, PathBuf},
     sync::{Arc, RwLock},
 };
-use wascc_codec::core::CapabilityConfiguration;
 
 mod chunks;
 
 #[cfg(not(feature = "static_plugin"))]
 capability_provider!(FileSystemProvider, FileSystemProvider::new);
 
-const CAPABILITY_ID: &str = "wascc:blobstore";
+const CAPABILITY_ID: &str = "wasmcloud:blobstore";
 const SYSTEM_ACTOR: &str = "system";
 const FIRST_SEQ_NBR: u64 = 0;
 const VERSION: &str = env!("CARGO_PKG_VERSION");
-const REVISION: u32 = 3; // Increment for each crates publish
+const REVISION: u32 = 4; // Increment for each crates publish
 
+#[derive(Clone)]
 pub struct FileSystemProvider {
     dispatcher: Arc<RwLock<Box<dyn Dispatcher>>>,
-    rootdir: RwLock<PathBuf>,
-    upload_chunks: RwLock<HashMap<String, (u64, Vec<FileChunk>)>>,
+    rootdir: Arc<RwLock<PathBuf>>,
```

Comment:
If we don't actually need to use an `Arc` here, we should probably strip this. The litmus test I use is if we never need to clone this lock (which is required to pass it across a thread boundary), then we don't need to use an Arc.

Reply:
The reason for the `Arc` is so the FileSystemProvider can easily `derive(Clone)`, `RwLock` doesn't allow for cloning

Reply:
Yeah I forgot that a while back I made it so that the capability provider trait implementers had to be clone-able. Move along, nothing to see here. Please disperse!